tests/strategies/market_agents/test_future_agent.py

```diff
@@ -48,12 +48,3 @@ def test__init__creates_engines_of_correct_type(future_agent: FutureAgent) -> No
         """Test whether __init__ creates list of engines off type FutureEngine."""
         for engine in future_agent.engines:
             assert isinstance(engine, FutureEngine)
-
-    @staticmethod
-    def test_event_market_cycle_calls_engine_clean_up_buffers(future_agent: FutureAgent) -> None:
-        """Test whether event_market_cycle calls clean_up_buffers of all engines."""
-        for engine in future_agent.engines:
-            engine.clean_up_order_buffers = Mock()
-        future_agent.event_market_cycle()
-        for engine in future_agent.engines:
-            engine.clean_up_order_buffers.assert_called_once()
```

Comment:
CAn you please explain? This method is still used, right?

Reply:
We already have several tests covering the cleanup behavior in `tests/strategies/market_agents/test_future_engine.py`,  `test_clean_up_order_buffers_removes_all_traces_of_unneeded_bids` , `test_clean_up_order_buffers_removes_all_traces_of_unneeded_offers`